### PR TITLE
support displaying metadata in YAML maps via dot notation

### LINKED ISSUE #530: [Feature]: Access metadata from YAML Maps
### Describe the bug

I have the following front matter in my movie notes and I'm trying to access fields inside info here. In Dataview `info.title` works as expected but not in Kanban.
```
aliases: ["About Time"]
tags: [film]
rating: 8
people:
dateCreated: 2020-11-14T23:47
info: 
  title: About Time
  year: 2013
  rated: R
  released: 2013-11-08
  runtime: 123
```

### Expected Behavior

In the above example, `info.title` must return `About Time` but it doesn't show anything. 

### Steps to reproduce

Add a `Linked Page Metadata` field using info.title from the above front matter. It wouldn't show up.

### Relevant errors (if available)

_No response_

### Operating system

macOS

### PR DESCRIPTION
**Pull request by GamerGirlandCo**
that line added to the gitignore was so that i could test without affecting my main vault. i can remove it if you want 😅
[closes #530]

### CODE REVIEW

File: src/parsers/common.ts [hunk 1]
```diff
@@ -160,13 +174,25 @@ export function getLinkedPageMetadata(
...
       metadata[k.metadataKey] = {
         ...k,
         value: cache.frontmatter[k.metadataKey],
       };
       haveData = true;
+    } else if (
+      typeof serializedObject === 'string' ||
+      Array.isArray(serializedObject)
+    ) {
+      console.log('dottoe', serializedObject);
```
Comment (kometenstaub) [inline src/parsers/common.ts:189]: Is there a reason for logging this?

Comment (GamerGirlandCo) [reply src/parsers/common.ts:189]: > Is there a reason for logging this?

my bad forgot to remove that 😅

**Comment by GamerGirlandCo** [comment]
~~happy 3 week limbo anniversary lol~~

**Comment by mgmeyers** [comment]
@GamerGirlandCo I slightly modified your approach and merged it in a separate commit. Closing this!

**Pull request closed without merging**